jwplayer-appletv-web-app/js/application.js

```diff
@@ -55,12 +55,29 @@ App.onLaunch = function(opts) {
 
 }
 
+App.onResume = function() {
+  reloadPlaylists();
+}
+
+function reloadPlaylists() {
+  PLAYLISTS = {};
+  MEDIA_ITEMS = {};
+  var templateLoader = new TemplateLoader();
+  templateLoader.load("templates/ListCollection.tvml", function (templateDoc) {
+    // We're assuming that the ListCollection is always on the bottom of the document stack.
+    // This may not be the smartest thing to do, ideally we would want to have some "tag" attached
+    // to documents that we can use to safely determine whether we are replacing the correct document.
+    var oldListCollectionDoc = navigationDocument.documents[0];
```

Comment:
You can check the document's `data-view` attribute to see if it's the ListCollection, e.g:

`navigationDocument.documents[0].firstChild.getAttribute("data-view")  // "ListCollection"`